_includes/how-it-works/how-it-works.scss

```diff
@@ -3,10 +3,14 @@
 .how-it-works-section {
   background-color: white;
 
+  @include small-tablet-screen {
+    padding: 122px 0 !important;
```

Comment:
Why `!important`?

Reply:
because we need to rewrite styles from style.scss